x-pack/metricbeat/modules.d/aws.yml.disabled

```diff
@@ -2,13 +2,13 @@
 # Docs: https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-module-aws.html
 
 - module: aws
-  period: 300s
+  period: 1m
```

Comment:
Same as above :) Why changing this to 1m?

Reply:
Same here